core/modules/filter/filter.pages.inc

```diff
@@ -175,7 +175,7 @@ function filter_format_editor_image_form($form, &$form_state, $format) {
     // Remember this is editing, not inserting. In the event the image is
     // removed and re-uploaded.
     $form_state['storage']['editing_image'] = TRUE;
-    backdrop_set_title(t('Edit image'));
+    backdrop_set_title(t('Edit image details'));
```

Comment:
Is this from a different PR you are working on seems unrelated to this issue?

Reply:
Sorry, my mistake, this was from a different PR, I didn't mean to include it in this PR.